src/error.rs

```diff
@@ -97,6 +97,11 @@ pub enum Error {
     /// The signature algorithm for a signature is not in the set of supported
     /// signature algorithms given.
     UnsupportedSignatureAlgorithm,
+
+    /// A iPAddress name constraint was invalid:
+    /// - it had a sparse network mask (ie, cannot be written in CIDR form).
+    /// - it was too long or short
+    InvalidNetworkMaskConstraint,
```

Comment:
Note, this is semver-incompatible. I think it's a good change, but probably good to bump the version in the same PR (maybe with an `-alpha.1` suffix?) to make that obvious. Notably, this might influence how we want to adapt the API to support IP addresses in #5.

Reply:
Worth noting that briansmith/webpki main that we branched from has some breaking changes already compared to 0.22.0 that rustls currently depends on. Aside from that, I was fully expecting the first rustls-webpki release to have breaking changes compared to webpki 0.22.0.

Once this and the ip address stuff is landed, I'll do a release preparation PR which will: back out 17d91899, set the version here to 0.123.0 (adding 100 to semver minor to hopefully separate webpki and rustls-webpki version spaces), and start a changelog in the README here.

> (We might want to make Error non_exhaustive so we can more easily add stuff here in the future?)

Will do that here.

Reply:
Ahh yeah, I forgot that this is effectively a new crate so semver doesn't even matter. Perhaps it makes sense to just call the initial release 0.100.0 then?

Note that, once we change rustls to depend on rustls-webpki instead of the webpki crate, that is a semver-incompatible change (due to https://github.com/rustls/rustls/pull/858) so in order to release that in rustls we'll want  to execute on all the known API-incompatible issues there before the next release.